src/cargo/lib.rs

```diff
@@ -41,6 +41,8 @@ use term::color::{BLACK};
 
 pub use util::{CargoError, CargoResult, CliError, CliResult, human, Config, ChainError};
 
+pub const CARGO_ENV: &'static str = "CARGO";
```

Comment:
I don't really have a strong feeling one way or another ([hehe](https://www.youtube.com/watch?v=ussCHoQttyQ)) but I guess, like you said, `CARGO` is good for consistency with `RUSTC` ...